src/frontend/src/server/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
server folder's own TypeScript config file to allow server code to use node API not available in the client code